src/main/resources/archetype-resources/silo/src/main/resources/application.yml

```diff
@@ -17,13 +17,14 @@ spring:
             charset: UTF-8
             enabled: true
             force: true
-    config:
```

Comment:
So there was actually no reason for `spring.config.name`?

Reply:
No, not as far as I know.